js/app.js

```diff
@@ -8,15 +8,15 @@ Product.allNames = []; //Holds all product names
 Product.allVotes = []; //Holds number of votes for all products
 Product.currentDisplay = []; //Holds index numbers of products that are currently displayed
 Product.indexHolder = []; //Holds index numbers to be compared to current display
-Product.ITERATIONS = 5; //Constant number of voting rounds
+Product.ITERATIONS = 25; //Constant number of voting rounds
 Product.NUMBER_DISPLAYED = 3; //Constant how many images are displayed per voting round
 Product.iterationCounter = 0; //Tracks number of iterations
 Product.choice; //Contains user's latest choice
 Product.firstImage = document.getElementById('firstImage'); //HTML location of first image
 Product.secondImage = document.getElementById('secondImage'); //HTML location of second image
 Product.thirdImage = document.getElementById('thirdImage'); //HTML location of third image
 Product.elementObject = document.getElementById('chooseProduct'); //HTML location of form
-Product.elementObject.addEventListener('submit', handleVote); //Create event listener
+Product.elementObject.addEventListener('click', handleVote); //Create event listener
```

Comment:
look out! if I very carefully put my mouse in between two images and click 25 times, I get 25 "votes" for no products :(